ydb/docs/en/core/reference/ydb-sdk/topic.md

```diff
@@ -1261,15 +1263,28 @@ Reading messages one-by-one is not supported in the C++ SDK. Class `TDataReceive
 
 ### Reading with consumer offset storage on the client side {#client-commit}
 
-When reading starts, the client code must transmit the starting consumer offset to the server:
+Instead of committing messages client application may track reading progress on its own. In this case client can provide a handler, which will be called back on each partition read start. This handler may set starting reading position for this partition.
```

Comment:
```suggestion
Instead of committing messages, the client application may track reading progress on its own. In this case, it can provide a handler, which will be called back on each partition read start. This handler may set a starting reading position for this partition.
```